crates/solidity/inputs/language/src/definition.rs

```diff
@@ -194,21 +194,6 @@ codegen_language_macros::compile!(Language(
                                     Atom("*")
                                 ]))
                             )]
-                        ),
-                        Keyword(
-                            name = SolidityKeyword,
-                            identifier = Identifier,
-                            definitions = [KeywordDefinition(value = Atom("solidity"))]
```

Comment:
Can we keep it here but add the Reserved = Never? It makes more sense to only list them under the context of pragma, since it is meaningfully only there.

Otherwise, LGTM, thanks!

Reply:
Done.